ansible/playbooks/roles/compatibility_layer/tasks/main.yml

```diff
@@ -17,7 +17,8 @@
     path: "{{ gentoo_prefix_path }}/startprefix"
   register: startprefix
 
-- include_tasks: install_prefix.yml
+- name: Include task install_prefix.yml
+  ansible.builtin.include_tasks: install_prefix.yml
```

Comment:
I guess this was changed for a similar reason as `community.general.portage` above?

Reply:
That was changed to make the sensible-linter happy.